This patch file is from Qualcomm Innovation Center, Inc. and is provided under the following license:

Copyright (c) 2025 Qualcomm Innovation Center, Inc. All rights reserved.
SPDX-License-Identifier: BSD-3-Clause-Clear

From 39296ef921cd0a8295798283a154f32e85f06021 Mon Sep 17 00:00:00 2001
From: Lakshman Chandu Kondreddy <quic_lkondred@quicinc.com>
Date: Fri, 14 Feb 2025 18:27:04 +0530
Subject: [PATCH] Add support for Adreno623 GPU

Add initial changes to enable basic functionality of Adreno623 GPU.

Upstream-Status: Submitted [https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/33525]
Signed-off-by: Lakshman Chandu Kondreddy <quic_lkondred@quicinc.com>
---
 src/freedreno/common/freedreno_devices.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/freedreno/common/freedreno_devices.py b/src/freedreno/common/freedreno_devices.py
index 169ada54545..e28296ece1a 100644
--- a/src/freedreno/common/freedreno_devices.py
+++ b/src/freedreno/common/freedreno_devices.py
@@ -502,6 +502,7 @@ add_gpus([
 
 add_gpus([
         GPUId(chip_id=0xffff06020100, name="FD621"),
+        GPUId(chip_id=0xffff06020300, name="Adreno623"),
     ], A6xxGPUInfo(
         CHIP.A6XX,
         [a6xx_base, a6xx_gen3, A6XXProps(lrz_track_quirk = False)],
-- 
2.34.1

